auslib/web/public/swagger/public_api_spec.yml

```diff
@@ -84,79 +84,6 @@ paths:
         '404':
           $ref: '#/responses/resourceNotFound'
 
-  /api/v1/releases/{release}/revisions:
```

Comment:
Yeah, I can see an argument that. As far as I know though, nobody looks at releases history through the public API though, perhaps we can defer this until it becomes needed?

Reply:
Looks like a whopping 7 requests for that endpoint since 2019-03-07, compared to 56K beginning with /api/v1/releases, so that seems fair to me. That's in the balrog_prod.balrog_elb_logs_aus_api data.